client/asset/btc/btc.go

```diff
@@ -660,6 +660,12 @@ func (btc *ExchangeWallet) feeRateWithFallback(confTarget uint64) uint64 {
 	return feeRate
 }
 
+type sat uint64
+
+func (s sat) String() string {
+	return strconv.FormatFloat(btcutil.Amount(s).ToBTC(), 'f', -1, 64) // dec, but no trailing zeros
+}
+
```

Comment:
@buck54321 BTW, I noticed in one of your PRs you made a bunch of changes to the logging in the same vein, although my approach with FormatFloat was so that the value is only displayed with as many digits as required.  That is 1.21 DCR would display as `1.21000000` with `%.8f`, but `1.21` with `FormatFloat` with the `-1` precision arg.  How do you feel about the display and removing the trailing zeros?  I feel like most of the time it hurts readability to keep the zeros.

Reply:
That's good. I'll adapt mine.